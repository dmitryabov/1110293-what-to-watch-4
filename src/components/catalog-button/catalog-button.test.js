import React from "react";
import renderer from "react-test-renderer";
import CatalogButton from "./catalog-button.jsx";


it(`Render WelcomeScreen`, () => {

  const tree = renderer
  .create(<CatalogButton
    clickOnButton={() => {}}
  />).toJSON();

  expect(tree).toMatchSnapshot();
});
