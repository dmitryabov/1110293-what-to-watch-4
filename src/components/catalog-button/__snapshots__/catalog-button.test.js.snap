// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render WelcomeScreen 1`] = `
<button
  className="catalog__button"
  onClick={[Function]}
  type="button"
>
  Show more
</button>
`;
